torch/csrc/api/include/torch/nn/functional/dropout.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <torch/nn/options/dropout.h>
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor alpha_dropout(const Tensor& input, const DropoutOptions& options, bool is_training) {
```

Comment:
I believe we can write
```suggestion
inline Tensor alpha_dropout(const Tensor& input, const AlphaDropoutOptions& options = {}, bool training = false) {
```
to match the Python names better and also more consistent with the naming of other functional options :D